<launch>
    <!-- Camera Parameters -->
    <arg name="camera_resolution" default="HD" /> <!-- Choose between: VGA, HD, FHD, 2K-->
    <arg name="depth_mode" default="PERFORMANCE" /> <!-- Choose between: PERFORMANCE, QUALITY, ULTRA, NEURAL-->
    <!-- ROS Topic Parameters -->
    <arg name="left_image_topic" default="/zed/left_image" />
    <arg name="right_image_topic" default="/zed/right_image" />
    <arg name="stereo_image_topic" default="/zed/stereo_image" />

    <!-- Node -->
    <node name="zed_py" pkg="image" type="zed_py.py" output="screen">
        <!-- Set Parameters -->
        <param name="camera_resolution" value="$(arg camera_resolution)" />
        <param name="depth_mode" value="$(arg depth_mode)" />
        <param name="left_image_topic" value="$(arg left_image_topic)" />
        <param name="right_image_topic" value="$(arg right_image_topic)" />
        <param name="stereo_image_topic" value="$(arg stereo_image_topic)" />
    </node>
</launch>